test/distributed/org/apache/cassandra/fuzz/sai/SingleNodeSAITestBase.java

```diff
@@ -62,11 +63,11 @@ public abstract class SingleNodeSAITestBase extends TestBaseImpl
     private static final int VALIDATION_SKIP = 739;
     private static final int QUERIES_PER_VALIDATION = 8;
 
-    private static final int FLUSH_SKIP = 1499;
-    private static final int COMPACTION_SKIP = 1503;
-    private static final int DEFAULT_REPAIR_SKIP = 6101;
+    private static final int FLUSH_SKIP = 2217;
+    private static final int COMPACTION_SKIP = 4435;
+    private static final int DEFAULT_REPAIR_SKIP = 8869;
```

Comment:
These are just reverting to skips closer to the original skips before I changed this in CASSANDRA-20189. We don't need to repair 5 times in 30,000 ops.